pkg/config/proxy.go

```diff
@@ -399,6 +400,7 @@ func (cfg *BaseProxyConf) unmarshalFromMsg(pMsg *msg.NewProxy) {
 	cfg.ProxyType = pMsg.ProxyType
 	cfg.UseEncryption = pMsg.UseEncryption
 	cfg.UseCompression = pMsg.UseCompression
+	cfg.BandwidthLimit, _ = NewBandwidthQuantity(pMsg.BandwidthLimit)
```

Comment:
will result in an empty bandwidth limit if `NewBandwidthQuantity(...)` returns an error

Reply:
Same for `bandwidth_limit_mode` below.  Defaults to `"client"` even if there is an error, so we can ignore and take the default if we want.